manifest/fr.baplar/ResoniteDiscordRpc/info.json

```diff
@@ -25,6 +25,21 @@
 					"installLocation": "/rml_libs"
 				}
 			]
+		},
+		"1.0.3": {
+			"releaseUrl": "https://github.com/Baplar/ResoniteDiscordRpc/releases/tag/v1.0.3",
+			"changelog": "- Recompiled for .NET 10\n- Fix disposal of existing official Discord RPC connector on Windows\n- Added optional debug logging",
+			"artifacts": [
+				{
+					"url": "https://github.com/Baplar/ResoniteDiscordRpc/releases/download/v1.0.3/ResoniteDiscordRpc.dll",
```

Comment:
Checking for `IsDebugEnabled()` for just a Debug log is a bit redundant, Debug logs are already only logged in debug mode. If you want to make sure you aren't doing extra processing for serializing something for said log, there is `DebugFunc(() => $"");` which will only run the string building or other method if it should be logged.